src/frontend/metrics/positionToCountries.ts

```diff
@@ -0,0 +1,29 @@
+import borders from '@osm_borders/maritime_10000m';
+import GeojsonGeometriesLookup from 'geojson-geometries-lookup';
+
+let lookup: undefined | GeojsonGeometriesLookup;
+
+export default function positionToCountries(
+  latitude: number,
+  longitude: number,
+): Set<string> {
+  lookup ??= new GeojsonGeometriesLookup(borders);
+
+  const result = new Set<string>();
+
+  const {features} = lookup.getContainers({
+    type: 'Point',
+    coordinates: [longitude, latitude],
```

Comment:
I thought this was backwards, but I believe it's right (and this function is unit tested).